src/player/Marine.cpp

```diff
@@ -72,3 +72,120 @@ int32_t Marine::checkForPickUp() {
     }
     return -1;
 }
+
+/**
+* Date: Mar 27 
+* Author: Aing Ragunathan
+*
+* Interface: void Marine::updateImageDirection()
+*
+* Description:
+*       This function changes the direction that the character is facing.
+*       It is called from GameStateMatch::handle after every frame and
+*       updates the direction of the sprite according to the angle of the
+*       mouse from the center of the screen.
+*/
+void Marine::updateImageDirection() {
+    double angle = getAngle();
+    double radians = (angle -90) * M_PI/180;
+    
+    //order: start from ~0 rad, counter clockwise
+    if (radians > -M_PI/8 && radians < M_PI/8) { 
+        Entity::setSpritePosY(SPRITE_SIZE_Y*2); //right
+    }
+    else if (radians > -3*M_PI/8 && radians < -M_PI/8) { 
+        Entity::setSpritePosY(SPRITE_SIZE_Y*3);  //back & right
+    }
+    else if (radians > -5*M_PI/8 && radians < -3*M_PI/8) { 
+        Entity::setSpritePosY(SPRITE_SIZE_Y*4);  //back
+    }
+    else if (radians > -7*M_PI/8 && radians < -5*M_PI/8) { 
+        Entity::setSpritePosY(SPRITE_SIZE_Y*5); //back & left
+    }
+    else if (radians > -9*M_PI/8 && radians < -7*M_PI/8) { 
```

Comment:
Magic numbers may produce wonder and amazement, but I am not a fan of magic shows
Please fix